### ISSUE TITLE
Contact form

### ISSUE THREAD

**Issue by crawfoal** [crawfoal/greensteps#165]
We were planning on having this send an to greenstepsinfo, but that might not be the best way to do things. I don't think we can just send an email from the user's email account on their behalf. We could send it from greenstepsinfo to greenstepsinfo. Or we could just display them in a "worklist" inside the admin section of the app.

**Comment by sam0208** [comment]
Hello, I am not really sure if you do not want to send e-mails on the user's behalf or that you can't do that.

For the latter you could try and send e-mails through an smtp server with their e-mail adres as the sender.

**Comment by crawfoal** [comment]
Wouldn’t that require their email password?

**Comment by sam0208** [comment]
No, smtp lets fill out all the e-mail information. This is frequently used by spammers to send e-mail with malware from an other adres. However it is also used for things like forwarding all e-mails form one account to another so that the sender's e-mail is correctly displayed in the other mailbox. 

https://en.wikipedia.org/wiki/Email_spoofing

**Comment by crawfoal** [comment]
Oh, good to know 🙂

Have you implemented something like this before? Does it cause the emails to go to the spam folder?

**Comment by sam0208** [comment]
No i have not but i think it would not be as difficult. If you are using rails mailers you can configure smtp settings in `config/environments/` easily. About the spam folder, I think you can whitelist e-mail coming from the server.

**Comment by crawfoal** [comment]
Cool, thanks for the information 🙂 

We are planning on including this in the MVP which has a deadline of May 31. Are you interested in working on this issue? If not we understand and appreciate your input!

**Comment by sam0208** [comment]
I would like to work on this issue but I'm a bit busy this month. I can check back sometime but I can't work on the issue right now.

**Comment by crawfoal** [comment]
Cool, we'll probably get to it by the end of the month but if not we'd love your help 🙂
